lib/parseMigrate.js

```diff
@@ -9,32 +9,40 @@ const promisifyWriteFile = promisify(fs.writeFile);
 
 module.exports = async filePath => {
   try {
-    const { stdout: docgenOut } = await promisifyExec(`react-docgen ${filePath} --pretty`);
+    const { stdout: docgenOut } = await promisifyExec(
+      `react-docgen ${filePath} --pretty --resolver findAllComponentDefinitions`
+    );
 
     // Success!
-    const object = await parseProps(JSON.parse(docgenOut));
-
+    const docgenOutArray = JSON.parse(docgenOut);
     console.log('🎉 completed: parsed React component', filePath);
 
-    const { stdout: ctfMigrateOut } = await promisifyExec(
-      `ctf-migrate create ${object.content_type} -c ${object.content_type}`
-    );
+    // Loop over array of components and create migration for each
+    docgenOutArray.forEach(async docgenOut => {
+      const object = await parseProps(docgenOut);
 
-    // Success!
-    const fileName = ctfMigrateOut.split('/')[ctfMigrateOut.split('/').length - 1].trim();
+      if (!object) return;
+
+      const { stdout: ctfMigrateOut } = await promisifyExec(
+        `ctf-migrate create ${object.content_type} -c ${object.content_type}`
+      );
+
+      // Success!
+      const fileName = ctfMigrateOut.split('/')[ctfMigrateOut.split('/').length - 1].trim();
 
-    console.log('🎉 completed: created migration', fileName);
+      console.log('🎉 completed: created migration', fileName);
 
-    if (isJSFile(fileName)) {
-      try {
-        await promisifyWriteFile(`./migrations/${object.content_type}/${fileName}`, createMigration(object));
-        console.log('All done!');
-      } catch (error) {
-        return console.log('fs error:', err);
+      if (isJSFile(fileName)) {
+        try {
+          await promisifyWriteFile(`./migrations/${object.content_type}/${fileName}`, createMigration(object));
+          console.log('All done!');
+        } catch (error) {
+          return console.log('fs error:', err);
```

Comment:
`console.error`?

Reply:
doesn't seem to make a difference in the terminal... but can update